book/website/project-design/sdp/personal.md

```diff
@@ -9,8 +9,23 @@ For example, a person’s name, address or other unique identifier such as their
 
 [“Data related to the deceased are not considered personal data in most cases under the GDPR.”](https://gdpr.eu/eu-gdpr-personal-data/) 
 
-Indirect identifiers include health, economic, cultural or social characteristics. 
-Especially when a certain combination of these identifiers can be used to identify a person, care must be taken to manage the data properly. 
+##### Direct identifiers
+
+["A person is directly identifiable if it’s possible to identify them using nothing but information (identifiers) at hand, controlled and processed, without introducing additional data from external sources (e.g. first and last name)."](https://infranet.hr/infranet-blog/direct-identification-of-an-individual-using-personal-data/)
```

Comment:
```suggestion
["A person is directly identifiable if it’s possible to identify them using nothing but information (identifiers) at hand, controlled and processed, without introducing additional data from external sources (such as first and last name)."](https://infranet.hr/infranet-blog/direct-identification-of-an-individual-using-personal-data/)
```

Reply:
Following our [style guide](https://deploy-preview-2580--the-turing-way.netlify.app/community-handbook/style.html#avoid-latin-abbreviation), we spell out these words because they're not screen reader friendly. For example, e.g. is read as egg.